src/visualization/include/iDynTree/Visualizer.h

```diff
@@ -466,6 +576,41 @@ class IModelVisualization
     virtual Transform getWorldLinkTransform(const LinkIndex& linkIndex) = 0;
 };
 
+class ITexture
```

Comment:
Can you add here or in some other related interface what you mean by "Texture"? In 3D Graphics texture is typically used to refer to 2D images that is then mapped to the surface of a 3D model to describe its appearance (see https://en.wikipedia.org/wiki/Texture_mapping). In this context instead if I understood correctly the texture is a sort of "render target" (see the second meaning in https://www.khronos.org/opengl/wiki/Texture). As this could be confusing for users, could you clarify this? Thanks!

Reply:
Yes, the correct interpretation would be that the texture is a different target for the renderer. Do you think that adding some documentation may be sufficient, or you about a better naming?

Reply:
As you prefer, I think that even just documentation is ok.

Reply:
I have updated the documentation in https://github.com/robotology/idyntree/pull/802/commits/b7267fc3ba69363f2fcadd9fc53f37e1dd845d7f